src/MongoDB/Driver/Manager.cpp

```diff
@@ -53,6 +53,7 @@ const StaticString
 	s_MongoDBDriverManager_readpreferencetags("readpreferencetags"),
 	s_MongoDBDriverManager_readPreference("readPreference"),
 	s_MongoDBDriverManager_readPreferenceTags("readPreferenceTags"),
+	s_MongoDBDriverManager_maxStalenessMS("maxStalenessMS"),
```

Comment:
It means I can't use the nice HHVM string lookups, and need to do a really annoying loop with conversions and strcasecmps. I am not intending to do that. I'm happy to add the check for all lower case though.